diff --git a/debian/control b/debian/control
index 57191e26..6f300aaf 100644
--- a/debian/control
+++ b/debian/control
@@ -61,7 +61,7 @@ Depends: ${misc:Depends},
   elbe-common (= ${binary:Version}),
   elbe-control (= ${binary:Version}),
   qemu-utils,
-  qemu-kvm (>= 1:2.1),
+  qemu-kvm,
   python-mako,
   p7zip-full,
   make
diff --git a/elbepack/init/Makefile.mako b/elbepack/init/Makefile.mako
index 21f21cd0..7aed5286 100644
--- a/elbepack/init/Makefile.mako
+++ b/elbepack/init/Makefile.mako
@@ -85,7 +85,6 @@ all: .stamps/stamp-install-initial-image
 .stamps/stamp-install-initial-image: .stamps/stamp-create-buildenv-img
 	@ echo $(INTERPRETER)
 	@ $(INTERPRETER) -M $(MACHINE) \
-		-device virtio-rng-pci \
 		-drive file=buildenv.img,if=$(HD_TYPE),bus=1,unit=0 \
 % if prj.has("mirror/cdrom"):
 		-drive file=${prj.text("mirror/cdrom")},if=$(CDROM_TYPE),media=cdrom,bus=1,unit=0 \
@@ -116,7 +115,6 @@ all: .stamps/stamp-install-initial-image
 
 run:
 	$(INTERPRETER) -M $(MACHINE) \
-		-device virtio-rng-pci \
 		-device virtio-net-pci,netdev=user.0 \
 		-drive file=buildenv.img,if=$(HD_TYPE),bus=1,unit=0 \
 		-no-reboot \
@@ -134,7 +132,6 @@ run:
 
 run-con:
 	$(INTERPRETER) -M $(MACHINE) \
-		-device virtio-rng-pci \
 		-device virtio-net-pci,netdev=user.0 \
 		-drive file=buildenv.img,if=$(HD_TYPE),bus=1,unit=0 \
 		-no-reboot \
