diff --git a/debian/control b/debian/control
index 158df1b..4662408 100644
--- a/debian/control
+++ b/debian/control
@@ -102,7 +102,7 @@ Depends: ${misc:Depends},
   python-elbe-bin (= ${binary:Version}),
   python-elbe-control (= ${binary:Version}),
   qemu-utils,
-  qemu-kvm (>= 1:2.1),
+  qemu-kvm,
   python-mako,
   p7zip-full,
   make
@@ -224,7 +224,7 @@ Depends: ${misc:Depends},
   python-suds,
   python-debian,
   python-elbe-common (= ${binary:Version}),
-  libvirt-clients,
+  libvirt-bin,
   libvirt-daemon-system
 Description: Commandline Tool to control an elbe buildenv
  The SOAP Interface is used to control an elbe buildenv.
@@ -236,7 +236,7 @@ Depends: ${misc:Depends},
   python3-suds,
   python3-debian,
   python3-elbe-common (= ${binary:Version}),
-  libvirt-clients,
+  libvirt-bin,
   libvirt-daemon-system
 Description: Commandline Tool to control an elbe buildenv
  The SOAP Interface is used to control an elbe buildenv.
diff --git a/elbepack/init/Makefile.mako b/elbepack/init/Makefile.mako
index c4f7957..3750db7 100644
--- a/elbepack/init/Makefile.mako
+++ b/elbepack/init/Makefile.mako
@@ -84,7 +84,6 @@ all: .stamps/stamp-install-initial-image
 .stamps/stamp-install-initial-image: .stamps/stamp-create-buildenv-img
 	@ echo $(INTERPRETER)
 	@ $(INTERPRETER) -M $(MACHINE) \
-		-device virtio-rng-pci \
 		-drive file=buildenv.img,if=$(HD_TYPE),bus=1,unit=0 \
 % if prj.has("mirror/cdrom"):
 		-drive file=${prj.text("mirror/cdrom")},if=$(CDROM_TYPE),media=cdrom,bus=1,unit=0 \
@@ -115,7 +114,6 @@ all: .stamps/stamp-install-initial-image
 
 run:
 	$(INTERPRETER) -M $(MACHINE) \
-		-device virtio-rng-pci \
 		-device virtio-net-pci,netdev=user.0 \
 		-drive file=buildenv.img,if=$(HD_TYPE),bus=1,unit=0 \
 		-no-reboot \
@@ -133,7 +131,6 @@ run:
 
 run-con:
 	$(INTERPRETER) -M $(MACHINE) \
-		-device virtio-rng-pci \
 		-device virtio-net-pci,netdev=user.0 \
 		-drive file=buildenv.img,if=$(HD_TYPE),bus=1,unit=0 \
 		-no-reboot \
